### PR TITLE
update adapters

### PR DESCRIPTION
**Pull request by MardaWang0518**
Before submitting a new pull request, please make sure:

- [ ] Test cases have been added/updated/passed for the code you will submit.
- [ ] Documentation has added or updated.
- [ ] Commit message is following the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters.
- [ ] Squash the repeat code commits, short patches are welcome.

### CODE REVIEW

**Comment by MardaWang0518** [comment]
adapter interfaces recommitted

File: framework/ohos/src/main/ets/hippy_framework/modules/nativemodules/audio/AudioPlayerModule.ets [hunk 1]
```diff
@@ -18,14 +18,101 @@
...
+  }
+
+  seekTo( newPosition:number){
+
+  }
+
+  pause(){
+    if (this.mMediaPlayer != null) {
+      this.mMediaPlayer.pause().then(() => {
+        console.info('Renderer pauseed');
```
Comment (zealotchen0) [inline framework/ohos/src/main/ets/hippy_framework/modules/nativemodules/audio/AudioPlayerModule.ets:98]: 这里使用 LogUtils 接口来打印日志

File: framework/ohos/src/main/ets/hippy_framework/adapter/storage/DefaultStorageAdapter.ets [hunk 2]
```diff
@@ -33,143 +34,115 @@ const COLUMN_KEY = 'storage_key'
...
+    LogUtils.i(TAG, `ResultSet column names: ${resultSet.columnNames}, column count: ${resultSet.columnCount}`);
+    let results = new Array<string>(resultSet.rowCount);
+    let i = 0;
+    while (resultSet.goToNextRow()) {
+      results[i] = resultSet.getString(resultSet.getColumnIndex(COLUMN_KEY));
+      i++;
+    }
+    resultSet.close();
+    return results
+    //       if (err) {
```
Comment (zealotchen0) [inline framework/ohos/src/main/ets/hippy_framework/adapter/storage/DefaultStorageAdapter.ets:143]: 确认不需要的代码后续可以删掉

File: framework/ohos/src/main/ets/hippy_framework/adapter/http/HttpTaskCallbackImpl.ets [hunk 3]
```diff
@@ -46,36 +48,37 @@ export class HttpTaskCallbackImpl implements HttpTaskCallback {
...
-      if (DefaultHttpAdapter.isGzipRequest(request)) {
-        // inputStream = new GZIPInputStream(inputStream);
-      }
-      this.mDataHolder.readResourceDataFromStream(inputStream);
-    } catch (e) {
-      this.mDataHolder.errorMessage = e.getMessage();
-      this.mCallback.onHandleCompleted();
-      return;
-    }
+    // let inputStream = response.getInputStream();
```
Comment (zealotchen0) [inline framework/ohos/src/main/ets/hippy_framework/adapter/http/HttpTaskCallbackImpl.ets:51]: 确认不需要的代码可以删除，后续可以根据git记录来查询

File: framework/ohos/src/main/ets/hippy_framework/adapter/http/DefaultHttpAdapter.ets [hunk 4]
```diff
@@ -23,170 +23,130 @@ import HashMap from '@ohos.util.HashMap';
...
-import connection from '@ohos.net.connection';
 import { HippyException } from '../../../support/common/HippyException';
 import { LogUtils } from '../../../support/utils/LogUtils';
 import { BusinessError } from '@kit.BasicServicesKit';
 import { HippyHttpResponse } from './HippyHttpResponse';
+import { HippyEngineContext } from '../../HippyEngineContext';
 import { http } from '@kit.NetworkKit';
-import { call } from '@kit.TelephonyKit';
 
 const TAG = 'DefaultHttpAdapter'
```
Comment (zealotchen0) [inline framework/ohos/src/main/ets/hippy_framework/adapter/http/DefaultHttpAdapter.ets:39]: 这里 TAG 可以考虑加个前缀，方便搜索定位

**Comment by MardaWang0518** [comment]
收到！

**Pull request merged**